docs/source/analysis.rst

```diff
@@ -0,0 +1,10 @@
+Analysis
+========
+
+Radial Distribution Function
+----------------------------
+
+.. toctree::
+  :maxdepth: 1
+
+  RDF <_autosummary/dynsight.analysis.RDF>
```

Comment:
Class names should use camel case. Eg: "CamelCase" not "CAMELCASE" or "camelcase".

Reply:
changed class in RadialDistributionFunction and methods in:
- compute 
- find_minima_point
- plot_rdf

is it ok? or better without "_"

Reply:
Yes that is good, but the class name "RDF" needs to change too.